runtime/fastly/builtins/backend.cpp

```diff
@@ -1466,11 +879,6 @@ bool Backend::constructor(JSContext *cx, unsigned argc, JS::Value *vp) {
       return false;
     }
     std::string cipher_spec(ciphers_chars.begin(), ciphers_chars.len);
-    if (!is_cipher_suite_supported_by_fastly(cipher_spec)) {
-      JS_ReportErrorNumberASCII(cx, FastlyGetErrorMessage, nullptr,
-                                JSMSG_BACKEND_CIPHERS_NOT_AVALIABLE);
```

Comment:
Can we test this error with this code path removed?

Reply:
The host does not yet validate the ciphersuite list, waiting on https://github.com/fastly/ExecuteD/issues/2390 and https://github.com/fastly/ExecuteD/issues/4205